llama-index-integrations/llms/llama-index-llms-text-generation-inference/llama_index/llms/text_generation_inference/base.py

```diff
@@ -62,7 +62,10 @@ class TextGenerationInference(FunctionCallingLLM):
     )
     max_tokens: int = Field(
         default=DEFAULT_NUM_OUTPUTS,
-        description=("The maximum number of tokens to generate."),
+        description=(
+            LLMMetadata.model_fields["context_window"].description
+            + " Maximum total tokens returned from TGI endpoint."
+        ),
```

Comment:
I'm not sure this change is correct? The description for the context window and max tokens are two different things 
- context window is the absolute max input size 
- max_tokens is the maximum number of tokens the model can output. This is used to make sure in our query engines that enough room is left in the prompt to generate `max_tokens` -- for example, this commonly defaults to 256 or 512. Most APIs today have a max of 8192 or so (i.e. anthropic)

Reply:
You are right, I misplaced the description change, it was supposed to be for context_window and not max_tokens